tests/GridFS/FunctionalTestCase.php

```diff
@@ -34,6 +35,13 @@ public function setUp(): void
         $this->filesCollection = $this->createCollection($this->getDatabaseName(), 'fs.files');
     }
 
+    public function tearDown(): void
+    {
+        StreamWrapper::setContextResolver('default', null);
```

Comment:
Where does "default" come from? In `StreamWrapperFunctionalTest::testStreamWithContextResolver()` you use "bucket".

Is there any reason to have this `tearDown()` logic in this file if aliases are never registered here? Might make more sense to clean up closer to the actual tests that create aliases.

I'd also be in favor of segregating all of the alias-related logic to its own test file, unless that'd make things harder to maintain.